galaxy_release_util/point_release.py

```diff
@@ -613,8 +619,21 @@ def build_and_upload(
     for package in packages:
         click.echo(f"Building package {package}")
         build_package(package)
+    return packages
+
+
+@group_options(galaxy_root_option, packages_option, no_confirm_option)
+@cli.command("build-and-upload", help="Build and upload packages")
+def build_and_upload(
+    galaxy_root: Path,
+    package_subset: List[str],
+    no_confirm: bool,
+):
+    packages: List[Package] = _build(galaxy_root, package_subset)
     if no_confirm or click.confirm("Upload packages ?"):
         for package in packages:
+            if package.name == "meta":
+                build_meta_dependencies(package, packages, get_root_version(galaxy_root))
```

Comment:
I think this should be inside `_build()` instead (before `build_package(package)`), because `upload_package()` just uploads the already built packages from `dist/` .